whist/core/game/game.py

```diff
@@ -1,25 +1,35 @@
 """One Game of whist"""
+import json
 from typing import Optional
 
+from pydantic import BaseModel
+
 from whist.core.game.hand import Hand
 from whist.core.game.play_order import PlayOrder
 from whist.core.game.player_at_table import PlayerAtTable
 from whist.core.scoring.score_card import ScoreCard
-from whist.core.scoring.team import Team
 from whist.core.user.player import Player
 
 
-class Game:
+class Game(BaseModel):
     """
     One Game of whist.
     """
 
-    def __init__(self, teams: list[Team]):
-        super().__init__()
-        self.play_order: PlayOrder = PlayOrder(teams)
-        self.win_score: int = 3
-        self.score_card: ScoreCard = ScoreCard()
-        self.current_hand: Optional[Hand] = None
+    play_order: PlayOrder
+    win_score: int = 3
+    score_card: ScoreCard = ScoreCard()
+    current_hand: Optional[Hand] = None
+
+    # pylint: disable=too-few-public-methods
+    class Config:
+        """
+        Enables to have non models as field types and sets the encoder for play order.
+        """
+        arbitrary_types_allowed = True
+        json_encoders = {
+            PlayOrder: lambda x: json.dumps(x, cls=PlayOrder.PlayOrderEncoder)
```

Comment:
this converts the PlayOrder to a JSON string for saving -> https://pydantic-docs.helpmanual.io/usage/exporting_models/#json_encoders

Reply:
Yes, you want me to add a comment?

Reply:
Thanks I hate it